mogp_emulator/MultiOutputGP.py

```diff
@@ -10,7 +10,12 @@
 from mogp_emulator.Priors import GPPriors
 from patsy import ModelDesc
 
-class MultiOutputGP(object):
+class MultiOutputGPBase(object):
```

Comment:
I guess we're not really making use anywhere of both MultiOutputGP and MultiOutputGP_GPU inheriting from a common base class - no shared implementations, and nothing that uses the base class, but I guess it doesn't do any harm, and does leave open possibilities for the future...